jsonrpc/filter_manager.go

```diff
@@ -296,37 +296,62 @@ func (f *FilterManager) dispatchEvent(evnt *blockchain.Event) error {
 		}
 	}
 
+	closedFilterIDs := make([]string, 0)
+
 	// flush all the websocket values
-	for _, filter := range f.filters {
+	for id, filter := range f.filters {
 		if filter.isWS() {
```

Comment:
Isn't it just easier to continue if it's _not_ a WS filter?

Reply:
Nice catch. I fixed at dcb2346b957ab539d3e52daf2a313d1fec859913